myapp/getClients.js

```diff
@@ -0,0 +1,17 @@
+const request = require('request');
+const app = require('./app.js');
+
+const getClients = function () { 
+  app.get('/clients', function (req, res) {
+    request('https://api.createsend.com/api/v3.1/clients.json', function (err, response, data) {
+      if(err) {
+        res.send(err);
```

Comment:
It is highly recommended to send the status code(2xx, 3xx, 5xx) as well along with your response.